force-app/main/default/classes/searchAndPick.cls

```diff
@@ -0,0 +1,27 @@
+global with sharing class searchAndPick {
+          private final Account person;
+          public searchAndPick(ApexPages.StandardController stdController){
+    this.person= (Account)stdController.getRecord();
+  }
+    
+     public list <Contact> ctt {get;set;}
+    public list <Contact> cttt {get;set;}
+    public list <Contact> ctct {get;set;}
+    public String searchKey {get;set;}
+     public String modKey {get;set;}
+    public searchAndPick( ) {
+    }
+    public void search(){
+        ctt = [SELECT Id, LastName, title, phone, Is_Primary_Contact__c from Contact where Accountid =:this.person.Id AND lastname like:'%'+searchKey+'%' LIMIT 10];
+}
```

Comment:
all queries and dml operations should be wrapped with try-catch blocks

Reply:
limiting with 10 will cause problems for data more than 10 records, please use pagination to display all records